composer.json

```diff
@@ -3,15 +3,10 @@
     "description":"Composer installer for Magento modules",
     "keywords":[ "composer-installer", "magento"],
     "minimum-stability":"stable",
+    "test_version": "dev-master",
```

Comment:
I don't think this needed. (added by tests ?)

Reply:
I added this. It's needed for tests
magento-composer-installer/tests/MagentoHackathon/Composer/Magento/FullStack/AbstractTest.php:21
magento-composer-installer/tests/MagentoHackathon/Composer/Magento/FullStack/AbstractTest.php:59

`perl -pi -e \'s/"version"/"test_version"/g\' ./composer.json`